cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/TopicReplicationFactorChangeParameters.java

```diff
@@ -36,6 +37,7 @@
     validParameterNames.add(REPLICATION_FACTOR_PARAM);
     validParameterNames.add(SKIP_RACK_AWARENESS_CHECK_PARAM);
     validParameterNames.add(CONCURRENT_PARTITION_MOVEMENTS_PER_BROKER_PARAM);
+    validParameterNames.add(MAX_PARTITION_MOVEMENTS_IN_CLUSTER_PARAM);
```

Comment:
Can we update the JavaDoc of `TopicConfigurationParameters` to show the use of this new parameter?